Brokerages/InteractiveBrokers/FinancialAdvisor/FinancialAdvisorConfiguration.cs

```diff
@@ -0,0 +1,161 @@
+﻿/*
+ * QUANTCONNECT.COM - Democratizing Finance, Empowering Individuals.
+ * Lean Algorithmic Trading Engine v2.0. Copyright 2014 QuantConnect Corporation.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+*/
+
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Threading;
+using System.Xml.Serialization;
+using IBApi;
+using QuantConnect.Brokerages.InteractiveBrokers.Client;
+using QuantConnect.Logging;
+
+namespace QuantConnect.Brokerages.InteractiveBrokers.FinancialAdvisor
+{
+    /// <summary>
+    /// Contains configuration data for a Financial Advisor
+    /// </summary>
+    public class FinancialAdvisorConfiguration
+    {
+        // Financial Advisor configuration data
+        private List<AccountAlias> _accountAliases = new List<AccountAlias>();
+        private List<Group> _accountGroups = new List<Group>();
+        private List<AllocationProfile> _allocationProfiles = new List<AllocationProfile>();
```

Comment:
nit - these properties should probably be made `readonly` - although, if not `readonly`, could instantiate a new instance instead of clearing them -- would remove the possibility of collection modified during enumeration error RE: line 48 and line 58. (Note sure if these are ever invoked from separate threads).

Reply:
The `Clear` and `Load` are always called sequentially by `Connect`, the `SubAccounts` property is not currently used.
I could have used `readonly`, but it would have required XML deserialization to temp objects followed by deep copy. Now the new instances returned from Deserialize are directly assigned to the members (line 131)